calculators/gas_king_rating.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+$GLOBALS['rating_calculators']['_mignoubou_gas_king'] = [
+    'description' => '
+        by <a href="/player/22267">PewPewPew</a>: gas king rating v2
```

Comment:
add a newline right after this subtitle, like done in other ratings

Reply:
Done